components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationMgtDBQueries.java

```diff
@@ -322,11 +330,27 @@ public class ApplicationMgtDBQueries {
 
     public static final String DELETE_SP_METADATA = "DELETE FROM SP_METADATA WHERE SP_ID = ?";
 
+    /**
+     * @deprecated Use ApplicationCertificateMgtService instead.
+     */
+    @Deprecated
     public static final String GET_CERTIFICATE_BY_ID = "SELECT CERTIFICATE_IN_PEM FROM IDN_CERTIFICATE WHERE ID = ?";
 
+    /**
+     * @deprecated Use ApplicationCertificateMgtService instead.
+     */
+    @Deprecated
     public static final String GET_CERTIFICATE_ID_BY_NAME = "SELECT ID FROM IDN_CERTIFICATE WHERE NAME = ? AND " +
             "TENANT_ID = ?";
+    /**
+     * @deprecated Use ApplicationCertificateMgtService instead.
+     */
+    @Deprecated
     public static final String REMOVE_CERTIFICATE = "DELETE FROM IDN_CERTIFICATE WHERE ID = ?";
+    /**
+     * @deprecated
```

Comment:
Let's mention that certificate removal for tenant is a back office process.

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/6087/commits/9a2dc889d25ec1c2d6d950643a6310f1717fd44a